earth_enterprise/src/common/ManifestEntry.h

```diff
@@ -29,8 +30,9 @@ struct ManifestEntry {
   std::string orig_path;
   std::string current_path;
   uint64 data_size;
+  std::vector<std::string> dependents;
```

Comment:
Should we add this to the `==` operator?

Reply:
Oh...  I missed that.  Yes it should be.